Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM golang:1.14 as builder
-RUN apt-get update && apt-get install -qyy liblzma-dev
+FROM golang:1.17 as builder
+RUN apt-get update && apt-get install -qyy liblzma-dev libssl-dev
```

Comment:
https://github.com/mendersoftware/mender-artifact/pull/287 <- not sure how relevant that discussion is by now @kacf @lluiscampos @oleorhagen, in any case this should be aligned with the `Dockerfile.binaries`

Reply:
done, thanks!